src/styles/utils/_loops.scss

```diff
@@ -0,0 +1,26 @@
+@for $day from 1 through 31 {
+  .p-#{$day}::before{
+    content:"#{$day}";
+  }
+}
+
+$days:
+  'mon' 0,
+  'tue' 1,
+  'wed' 2,
+  'thu' 3,
+  'fri' 4,
+  'sat' 5,
+  'sun' 6;
+
+@each $day, $index in $days {
+  .calendar--start-day--#{$day} > :first-child {
+    margin-left: #{($cell-side * $index) + ($cell-gap * (($index * 2) + 1))};
```

Comment:
This formula is right but could have been simple.

Reply:
Switched margin to the gap and simplified the formula.